src/tags/options.js

```diff
@@ -13,6 +13,7 @@ export const nullOptions = { nullStr: 'null' }
 
 export const strOptions = {
   defaultType: Type.PLAIN,
+  defaultQuote: Type.QUOTE_DOUBLE,
```

Comment:
Given that this only has two valid values, wouldn't it make sense to formulate it as a boolean instead?